examples/advanced-project/src/__checks__/synthetics/08-playwright-techniques.check.ts

```diff
@@ -0,0 +1,18 @@
+import * as path from 'path'
+import { BrowserCheck } from 'checkly/constructs'
+import { syntheticGroup } from '../utils/website-groups.check'
+
+// Even though there are multiple test() function calls in the .spec
+// file, they will all run and report as a single Checkly monitor. 
+// This check is deactivated, run `npx checkly test --update-snapshots` first. 
```

Comment:
This sentence is lacking context. Why is it deactivated? Why should I run `--update-snapshots`?

Reply:
added more explanation.